programs/rewards-oracle/src/instructions/set_current_rewards_wrapper_v0.rs

```diff
@@ -0,0 +1,73 @@
+use anchor_lang::prelude::*;
+use helium_entity_manager::state::*;
+use lazy_distributor::state::*;
+use lazy_distributor::{
+  cpi::{accounts::SetCurrentRewardsV0, set_current_rewards_v0},
+  LazyDistributor, SetCurrentRewardsArgsV0,
+};
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct SetCurrentRewardsWrapperArgsV0 {
+  pub entity_key: Vec<u8>,
+  pub oracle_index: u16,
+  pub current_rewards: u64,
+}
+
+#[derive(Accounts)]
+#[instruction(args: SetCurrentRewardsWrapperArgsV0)]
+pub struct SetCurrentRewardsWrapperV0<'info> {
+  #[account(mut)]
+  pub payer: Signer<'info>,
```

Comment:
Isn't actually payer, I'd call this oracle.

Reply:
True